Objects/longobject.c

```diff
@@ -159,6 +162,34 @@ _PyLong_New(Py_ssize_t size)
     return result;
 }
 
+/* Like _PyLong_New, but it uses `PyObject_Calloc` instead */
+PyLongObject *
+_PyLong_NewFillZero(Py_ssize_t size)
+{
+    PyLongObject *result;
```

Comment:
What would you think about removing this function, replacing the two calls to `_PyLong_NewFillZero(x)` with `_PyLong_New(x); memset(x->ob_digit, (x-1) * sizeof(digit))`?

Reply:
> What would you think about removing this function, replacing the two calls to `_PyLong_NewFillZero(x)` with `_PyLong_New(x); memset(x->ob_digit, (x-1) * sizeof(digit))`?

Shouldn't that be slower than just using `PyObject_Calloc`?